vimrc

```diff
@@ -272,14 +272,14 @@ endfunction
 command! -nargs=0 GitGrepWord :call GitGrepWord()
 nnoremap <silent> <Leader>gw :GitGrepWord<CR>
 
-function! GitWebURL() range
+function! GitHubURL() range
   let repo = systemlist("git config --get remote.origin.url | sed 's/\.git$//' | sed 's_^git@\\(.*\\):_https://\\1/_'")[0]
```

Comment:
On my local repo, I have the braintreeps repo as an upstream remote:

```
[remote "origin"]
  url = git@github.com:parrish/vim_dotfiles.git
  fetch = +refs/heads/*:refs/remotes/origin/*
[remote "upstream"]
  url = https://github.com/braintreeps/vim_dotfiles.git
  fetch = +refs/heads/*:refs/remotes/upstream/*
[branch "git_web_url"]
  remote = upstream
  merge = refs/heads/git_web_url
```

So, I took a stab at getting it use the current remote:

```
let repo = systemlist("git remote get-url `git config branch.$(git name-rev --name-only HEAD).remote` | sed 's/\.git$//' | sed 's_^git@\\(.*\\):_https://\\1/_'")[0]
```

There's probably a more elegant way to do that though

Reply:
Good call. Some of our machines seem to have an older version of git without `git remote get-url` so I solved it a different way: 33d63d1